completion/available/dotnet.completion.bash

```diff
@@ -2,17 +2,13 @@
 about-completion "bash parameter completion for the dotnet CLI"
 # see https://docs.microsoft.com/en-us/dotnet/core/tools/enable-tab-autocomplete#bash
 
-_dotnet_bash_complete()
-{
-  local word=${COMP_WORDS[COMP_CWORD]}
+function _dotnet_bash_complete() {
+	local cur="${COMP_WORDS[COMP_CWORD]}" IFS=$'\n'
+	local candidates
 
-  local completions
-  completions="$(dotnet complete --position "${COMP_POINT}" "${COMP_LINE}" 2>/dev/null)"
-  if [ $? -ne 0 ]; then
-    completions=""
-  fi
+	read -d '' -ra candidates < <(dotnet complete --position "${COMP_POINT}" "${COMP_LINE}" 2> /dev/null)
```

Comment:
```suggestion
	read -d '' -ra candidates < <(dotnet complete --position "$COMP_POINT" "$COMP_LINE" 2> /dev/null)
```

Reply:
This is correct either way in this scenario; it's never wrong to enclose parameter names in curly braces, but it's wrong more often than you'd think not to, so I just rent to always type them. It's clearer visually and much less likely I'll forget them when it actually matters.